python/tk_unreal/unreal_sg_engine.py

```diff
@@ -83,10 +84,10 @@ def _get_context_url(self, engine):
         selected_asset = self.selected_assets[0] if self.selected_assets else None
         try:
             selected_actors = self.get_selected_actors()
-        except:
+        except Exception:
```

Comment:
is it something you don't want to log?

Reply:
I don't know, I just fixed the bare `except` which wouldn't pass flake8.